src/lib/sysfs.h

```diff
@@ -25,7 +25,9 @@
 #include "list.h"
 #include "status.h"
 
-#define SYSFS_PCI_DEVICES       "/sys/bus/pci/devices"
+#define SYSFS_PCI_DEVICES	"/sys/bus/pci/devices"
+
```

Comment:
probably not necessary empty line

Reply:
I made in on purpose because those defines has nothing common, but together could confuse reader (prefix SYSFS, SYSTEM starts same).